web/src/components/drawer/record/record-field.tsx

```diff
@@ -470,6 +475,22 @@ export const RecordField: React.FC<RecordFieldProps> = ({
         }
         return singleContainer(child);
       }
+      case ColumnsId.tcpflags: {
+        let child = emptyText();
+        if (typeof value === 'number' && !isNaN(value)) {
+          const serviceClassName = gettcpFlagsServiceClassName(value);
```

Comment:
If I read correctly, this wouldn't decompose the OR-combination, e.g. we can have 3 that is 1+2 = FIN+SYN

Reply:
ebpf agent doesn't generate this combo here is what we expect to see in the flags field today https://github.com/netobserv/netobserv-ebpf-agent/blob/main/bpf/utils.h#L11

Reply:
I updated the PR to bring in more combos but not the entire list as there some flags with less interest to decode but if I am wrong and we ended up we need to decode all possible combo I think we can revisit and the missing ones ?

Reply:
Hey @msherif1234  .. sorry I missed these comments.
I didn't intend to say to write down the full list of possible combinations, but to replace that with a func that's able to decompose the bitfields into the list of atomic values. Just like I did here in excel : https://docs.google.com/spreadsheets/d/1CHfUQ8Sq0JyE_4ab-x1Aza8HobXXH8MmqI09Q0kLGkI/edit?gid=0#gid=0 :)
I'll open a follow-up

Currently, many aggregated flags aren't decomposed, e.g:
![image](https://github.com/user-attachments/assets/d518538f-7aab-48c2-9764-79f70c7df110)